packages/components/src/components/common/inputtag/inputTag.tsx

```diff
@@ -0,0 +1,61 @@
+import { forwardRef, type ElementRef } from 'react';
+import { useField } from '../field/field';
+import { IvyIcon } from '../icon/icon';
+import './inputTag.css';
+
+import { type IvyIcons } from '@axonivy/ui-icons';
+
+export type InputTagProps = {
+  tags?: string[];
+  lines?: string[][];
```

Comment:
this will not be the input I think.
The input is just a string, single line for inputs, multiline for areas.

And the tagOrText function (which I would call `findTags` or something like that), should find all tags in it

Reply:
E.g. the text `This is a <%= ivy.log.info() %> text with a tag` should then result in `This is a [ivy tag] text`.
Of course also with multiple tags...